usaspending_api/awards/tests/test_awards_v2.py

```diff
@@ -281,6 +460,41 @@ def test_award_endpoint_generated_id(client, awards_and_transactions):
     assert json.loads(resp.content.decode("utf-8")) == expected_response_cont
 
 
+def test_award_mulitple_cfdas(client, awards_and_transactions):
+
+    resp = client.get("/api/v2/awards/3/")
+    print(json.loads(resp.content.decode("utf-8"))["cfda_info"])
+    assert resp.status_code == status.HTTP_200_OK
+    assert json.loads(resp.content.decode("utf-8"))["cfda_info"] == [
+        {
+            "cfda_objectives": "objectives",
+            "cfda_number": "10.002",
+            "cfda_title": "CFDA Title 2",
+            "federal_action_obligation_amount": 500.0,
+            "non_federal_funding_amount": 0.0,
+            "total_funding_amount": 500.0,
+            "cfda_federal_agency": "Agency 2",
+            "cfda_website": "www.website.biz",
+            "sam_website": "www.website.com",
+            "cfda_obligations": "whatever",
+            "cfda_popular_name": "Popular",
+        },
+        {
+            "cfda_objectives": "objectives",
+            "cfda_number": "10.001",
+            "cfda_title": "CFDA Title",
+            "federal_action_obligation_amount": 100.0,
+            "non_federal_funding_amount": 0.0,
+            "total_funding_amount": 100.0,
+            "cfda_federal_agency": "Agency 1",
+            "cfda_website": "www.website.biz",
+            "sam_website": "www.website.com",
+            "cfda_obligations": "whatever",
```

Comment:
Is this supposed to be a string? This smells like a number to me...

Reply:
it's not